internal/manager/backup.go

```diff
@@ -173,7 +173,6 @@ func (m *Manager) updateBackupLogs(v *volume.Volume, agentOutput utils.MsgFormat
 	}
 
 	v.LastBackupDate = time.Now().Format("2006-01-02 15:04:05")
-	v.Metrics.LastBackupDate.SetToCurrentTime()
```

Comment:
I must admit this part is not clear... Should the `LastBackupDate` be the date of the last backup process or the date of the latest snapshot? I think it was deliberate to not update the `LastBackupDate` metric but I can't remember why and, for now, I don't have any good reason not to.

Reply:
I think this way we get the last **successful** backup date

Reply:
Nope, you set the last backup date to the current time regardless of the backup status.